.murdock

```diff
@@ -161,8 +161,7 @@ _greplist() {
 
 # get list of all app directories
 get_apps() {
-    make -f makefiles/app_dirs.inc.mk info-applications \
-        | $(_greplist $APPS) | sort
+    examples/suit_update tests/riotboot tests/riotboot_flashwrite
```

Comment:
this would need to be echoed, one per line.
you could also just set `APPS="examples/suit_update tests/riotboot tests/riotboot_flashwrite"' somewhere up.

Reply:
Thanks!